### PR TITLE
[docs] Add status header for coralogix exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for coralogix exporter,pls review:)

### CODE REVIEW

File: exporter/coralogixexporter/README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Coralogix Exporter
 
+| Status                   |              |
+| ------------------------ |--------------|
+| Stability                | [beta]       |
+| Supported pipeline types | traces, logs |
+| Distributions            | [contrib]    |
```
Comment (codeboten) [inline exporter/coralogixexporter/README.md:7]:
```suggestion
| Status                   |                 |
| ------------------------ |-----------------|
| Stability                | [beta]          |
| Supported pipeline types | traces, metrics |
| Distributions            | [contrib]       |
```

**Review by codeboten — COMMENTED** [review]
Pinging codeowners for confirmation of stability status  @oded-dd @ofirshmuel

**Comment by Frapschen** [comment]
@oded-dd @ofirshmuel can you review this PR? :)

**Comment by mx-psi** [comment]
@oded-dd @ofirshmuel PTAL!

**Review by oded-dd — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Comment by mx-psi** [comment]
@Frapschen Could you merge main into your branch? Otherwise tests won't pass

**Review by codeboten — APPROVED** [review]

**Pull request merged**